### PR TITLE
Refactor/update back and continue buttons

### PR DESCRIPTION
**Pull request by myang1010**
What (if any) features are you implementing?
- [Update "Back" and "Continue" buttons](https://github.com/Gary-Community-Ventures/benefits-calculator/issues/686)

What (if anything) did you refactor?
- Include "Open Sans" font family
- Float Back and Continue buttons left and right
- Update App.css file and add the new css style
- Apply the new css style to PreviousButton and ContinueButton

Any other comments, questions, or concerns?
- <strike>Seems if we have translations setup correctly, we have to Edit/Update `previousButton` in Translations. We also need to make updates for other supported languages.</strike>

Screenshot
- Float "Back" and "Continue" button left and right
>Screenshot 2023-12-06 at 9 18 52 AM

- Hover on "Back" button
>Screenshot 2023-12-06 at 9 22 47 AM

-Hover on "Continue" button
>Screenshot 2023-12-06 at 9 22 59 AM

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -34,6 +34,43 @@
...
+.prev-save-continue-buttons .back-button:hover {
+  background-color: transparent;
+}
+
+.nav-btn.continue-button,
+.back-continue-buttons .continue-button,
+.prev-continue-results-buttons .continue-button,
+.prev-save-continue-buttons .continue-button {
+  font-family: 'Open Sans';
+  background-color: #293457;
```
Comment (CalebPena) [inline src/App.css:59]: Update these colors when https://github.com/Gary-Community-Ventures/benefits-calculator/issues/765 is completed.

File: src/App.css [hunk 2]
```diff
@@ -34,6 +34,43 @@
...
 }
 
+.nav-btn.back-button,
+.back-continue-buttons .back-button,
+.prev-continue-results-buttons .back-button,
+.prev-save-continue-buttons .back-button {
+  font-family: 'Open Sans';
+  color: #41528c;
+  background-color: transparent;
+  font-weight: 700;
```
Comment (CalebPena) [inline src/App.css:44]: It looks like the buttons on the results page are no longer bold.

**Review by CalebPena — COMMENTED** [review]
Looks good, just a couple small comments. @sydney-devine What do you think about adding a hover effect on the `back` button. @myang1010 You can change the translations locally by going to http://localhost:8000/api/translations/admin, searching the message id in the label field, clicking edit, editing the `en-us` option, and clicking save. You can also automatically translate it into other languages by pressing the translate button if it does not happen automatically.

**Comment by sydney-devine** [comment]
> Looks good, just a couple small comments. @sydney-devine What do you think about adding a hover effect on the `back` button. @myang1010 You can change the translations locally by going to http://localhost:8000/api/translations/admin, searching the message id in the label field, clicking edit, editing the `en-us` option, and clicking save. You can also automatically translate it into other languages by pressing the translate button if it does not happen automatically.

I think a hover makes sense. However, the font on that is the mid blue I think. We're doing lavender on the other hover, but I think that might make the text too light. . . Open to suggestions on it. 

Can I get an updated screenshot of this?

**Comment by myang1010** [comment]
> I think a hover makes sense. However, the font on that is the mid blue I think. We're doing lavender on the other hover, but I think that might make the text too light. . . Open to suggestions on it.
> 
> Can I get an updated screenshot of this?

Just an idea for hovering: consider using a lavender background or leaving it transparent?

> bg_lavender bg_transparent

And no hovering,

> no_hover

**Comment by sydney-devine** [comment]
> > I think a hover makes sense. However, the font on that is the mid blue I think. We're doing lavender on the other hover, but I think that might make the text too light. . . Open to suggestions on it.
> > Can I get an updated screenshot of this?
> 
> Just an idea for hovering: consider using a lavender background or leaving it transparent?
> 
> > bg_lavender bg_transparent
> 
> And no hovering,
> 
> > no_hover

Let's do the hover but without the grey border. 

For the "<" part of the back button, can we make it so it isn't so wide? Not sure if we need to pull an icon for this, but would like for it to be a bit smaller more similar to the comp. It just looks a little awkward right now.

**Comment by myang1010** [comment]
> 
> Let's do the hover but without the grey border.
> 

Hey @sydney-devine, just want to confirm that we want the one with lavender background without border? Something like this? 
>new_hover

> You can use [this icon](https://mui.com/material-ui/material-icons/?query=arrow&selected=NavigateBefore). React MUI has an [easy way to add icons to buttons](https://mui.com/material-ui/react-button/#buttons-with-icons-and-label).

And thank you @CalebPena for the resource!

**Comment by sydney-devine** [comment]
> > Let's do the hover but without the grey border.
> 
> Hey @sydney-devine, just want to confirm that we want the one with lavender background without border? Something like this?
> 
> > new_hover
> 
> > You can use [this icon](https://mui.com/material-ui/material-icons/?query=arrow&selected=NavigateBefore). React MUI has an [easy way to add icons to buttons](https://mui.com/material-ui/react-button/#buttons-with-icons-and-label).
> 
> And thank you @CalebPena for the resource!

@myang1010  - can we try this hex code for the hover highlight? #E6E7E8. I think the lavender is too dark with the mid blue. Can also try this #FBF9FC

**Comment by myang1010** [comment]
> @myang1010 - can we try this hex code for the hover highlight? #E6E7E8. I think the lavender is too dark with the mid blue. Can also try this #FBF9FC

Okay, here's how they look,
- #E6E7E8
>Screenshot 2023-11-30 at 2 00 04 PM
- And #E6E7E8
> Screenshot 2023-11-30 at 2 02 21 PM

The first one "#E6E7E8" is more obvious,  and the second one is a little difficult to notice.

**Comment by sydney-devine** [comment]
@myang1010 Per slack thread, let's go with the original lavender 
image

File: src/App.css (hunk 2 above)
Comment (myang1010) [reply src/App.css]: Okay, switch font-weight to `bold`.

**Review by CalebPena — CHANGES_REQUESTED** [review]
Could you pull from `Marianna/734/typography-updates` and use the updated style controller. Also change the base branch to `Marianna/734/typography-updates`

**Comment by myang1010** [comment]
> Could you pull from `Marianna/734/typography-updates` and use the updated style controller. Also change the base branch to `Marianna/734/typography-updates`

Okay, pull in the changes from `Marianna/734/typography-updates` and retargeting the base branch to the same one.

File: src/App.css [hunk 3]
```diff
@@ -46,6 +46,26 @@
   margin-bottom: 1rem;
 }
 
+.nav-btn.back-button,
+.back-continue-buttons .back-button,
+.prev-continue-results-buttons .back-button,
+.prev-save-continue-buttons .back-button {
+  font-family: 'Open Sans';
+  color: #41528c;
```
Comment (CalebPena) [inline src/App.css:54]: Can you use the style controller for this?

File: src/App.css [hunk 4]
```diff
@@ -46,6 +46,26 @@
...
+  background-color: transparent;
+  font-weight: bold;
+  border: none;
+}
+
+.nav-btn.back-button:hover,
+.back-continue-buttons .back-button:hover,
+.prev-continue-results-buttons .back-button:hover,
+.prev-save-continue-buttons .back-button:hover {
+  background-color: #ecdeed;
```
Comment (CalebPena) [inline src/App.css:64]: Also here

**Review by CalebPena — CHANGES_REQUESTED** [review]
Just 2 more places to use the style controller.

**Review by CalebPena — APPROVED** [review]
Looks good TY.

**Comment by sydney-devine** [comment]
@CalebPena @myang1010 In staging, the "BACK" is still staying "PREV"

Also, can we increase the padding between whatever the input above the buttons is and the buttons themselves? Add another 50% of whatever is there. The buttons just seem really close to the inputs right now. Amount of spacing on Step 1 (see screenshot) is good reference. 

image

image

**Pull request merged**